addons/mail/static/src/components/message/message.scss

```diff
@@ -132,110 +106,25 @@
 // ------------------------------------------------------------------
 
 .o_Message {
-    background-color: $white;
     transition: background-color .5s ease-out;
 
-    .o_Message_partnerImStatusIcon {
-        color: $white;
-    }
-
-    &.o-not-discussion {
-        background-color: gray('100');
-
-        &.o_Message_active {
-            background-color: mix(gray('100'), gray('200'));
-        }
-
-        .o_Message_partnerImStatusIcon {
-            color: gray('100'); // same color as background of parent
-        }
-
-        &.o-selected {
-            border-bottom: 1px solid darken(gray('400'), 5%);
-        }
-    }
-
-    &.o-selected {
-        background-color: gray('400');
-
-        .o_Message_partnerImStatusIcon {
-            color: gray('400');
-        }
-    }
-
-    &.o-has-message-selection:not(.o-selected) {
-        opacity: 0.5;
+    &.o-not-discussion.o-selected {
+        border-bottom: 1px solid darken(gray('400'), 5%);
     }
 
     &.o-highlighted {
         background-color: rgba($o-brand-primary, .1);
-    }
-
-    &.o_Message_active {
-        background-color: gray('100');
-    }
-}
-
-.o_Message_authorName {
-    font-weight: $font-weight-bold;
-}
-
-.o_Message_authorRedirect {
-    cursor: pointer;
-}
-
-.o_Message_content .o_mention {
```

Comment:
It seems that this class is no longer used.

Reply:
IIRC, `o_mention` class was forged during a message post, so that all "mentions" (`@`) had this classname.
Looks like `o_mail_redirect` and `o_channel_redirect` are now used, and no more `o_mention`:
https://github.com/odoo/odoo/blob/master/addons/mail/static/src/models/composer_view.js#L1166

It's possible that this style rule is applicable for older messages that were posted with old mention.
But I don't think it's that problematic for (very) old messages, as mentions are super relevant now and soon after, but becomes quickly irrelevant with time.

It's a bit sad that we don't have good way to upgrade old messages. It's something we are aware and will make code design decisions to solve this problem once and for all, without hacks here and there.